src/pages/RepoPage/PullsTab/PullsTable/PullsTable.spec.tsx

```diff
@@ -280,28 +261,12 @@ describe('PullsTable', () => {
       expect(patch).toBeInTheDocument()
     })
 
-    it('renders change column', async () => {
-      const { queryClient } = setup({})
-      render(<PullsTable />, { wrapper: wrapper(queryClient) })
-
-      const change = await screen.findByText('0.00%')
-      expect(change).toBeInTheDocument()
-    })
-
-    it('renders coverage column', async () => {
-      const { queryClient } = setup({})
-      render(<PullsTable />, { wrapper: wrapper(queryClient) })
-
-      const coverage = await screen.findByText('75.00%')
-      expect(coverage).toBeInTheDocument()
-    })
-
     describe('bundle analysis is enabled', () => {
       it('renders bundle analysis column', async () => {
         const { queryClient } = setup({ bundleAnalysisEnabled: true })
         render(<PullsTable />, { wrapper: wrapper(queryClient) })
 
-        const bundleAnalysis = await screen.findByText('Upload: ✅')
+        const bundleAnalysis = await screen.findByText('Upload: ⏳')
```

Comment:
is the default this hourglass icon now?

Reply:
No, not the default value. It's just reflective of the API response that's being mocked.